### ISSUE TITLE
An error occurred while uploading the file 

### ISSUE THREAD

**Issue by binbinshi** [minio/console#1413]
When uploading a large file of 20M through the browser, the error is: An error occurred while uploading the file.
![image](https://user-images.githubusercontent.com/17977730/150091395-358451de-78c0-4899-ba76-b43fec571207.png)

**Comment by prakashsvmx** [comment]
Could you please share the trace ?: 
start `mc admin trace <alias>`  just before uploading the file and stop when the error is shown.

**Comment by harshavardhana** [comment]
There will be something specific in the server logs as well..

**Comment by dvaldivia** [comment]
Are you accessing the Console behind some nginx?

**Comment by binbinshi** [comment]
报了499的错误
![image](https://user-images.githubusercontent.com/17977730/150282305-ced59ded-f7ec-4293-bd87-290c9e06d8ef.png)

**Comment by binbinshi** [comment]
http code  499 ,  nginx is not used 
mc trace -a -v minio

![image](https://user-images.githubusercontent.com/17977730/150283206-151dced8-d8eb-4da6-a3fe-99bcdc60f555.png)

**Comment by vadmeste** [comment]
> this means console client time out ?

Yes

**Comment by harshavardhana** [comment]
Yeah, it's just a timeout uploading from the client. Check if you are going through some another proxy here that is adding read socket timeouts here @binbinshi

**Comment by binbinshi** [comment]
i just run a minio server , doesn't have any other proxy ; 

start cmd : ./minio server /mnts/data/minio --console-address :9001  

is console client have some timeouts config ?

**Comment by harshavardhana** [comment]
> i just run a minio server , doesn't have any other proxy ;
> 
> start cmd : ./minio server /mnts/data/minio --console-address :9001
> 
> is console client have some timeouts config ?
> 
> update minio to the newest version , Server `minio` already running the most recent version 2022-01-08T03:11:54Z of MinIO error still： Client disconnected before response was ready

Yeah this looks like a console connection read timeout of some sort. Let's check the timeouts in the code in this path.
